makeflow/src/makeflow.c

```diff
@@ -2065,6 +2091,12 @@ if (enforcer && wrapper_umbrella) {
 
 	if(local_queue)
 		batch_queue_delete(local_queue);
+	
```

Comment:
Setting options before delete is unusual enough that an explanatory comment is needed here.